ext/Client/__init__.lua

```diff
@@ -121,7 +122,7 @@ function VEManagerClient:EnablePreset(p_ID)
 	end
 end
 
----@param p_ID string|nil
+---@param p_ID string
 function VEManagerClient:DisablePreset(p_ID)
```

Comment:
This needs to clear self.m_Lerping[p_ID] else it will just continue pulsing

Reply:
DisablePreset calls _DestroyVE, which does that